Darwin/Examples/StressTestApp/StressTestApp/RemoteMongoCollection+SynchronousCRUD.swift

```diff
@@ -0,0 +1,194 @@
+// swiftlint:disable force_try
+import MongoSwift
+import StitchCore
+import StitchCoreSDK
+@testable import StitchCoreRemoteMongoDBService
+@testable import StitchRemoteMongoDBService
+
+private let waitTimeout = UInt64(1e+10)
+
+private class SynchronizedDispatchDeque {
```

Comment:
[minor] ditto about duplicating all this logic. This should be shared in some utility module